sched/Kconfig

```diff
@@ -1937,3 +1937,13 @@ config GROUP_KILL_CHILDREN_TIMEOUT_MS
 		> 0 means wait timeout
 		= 0 means don't do kill signal
 
+config INITIAL_PID_LIMIT
```

Comment:
PIDHASH_INITIAL_COUNT or PIDHASH_INITIAL_NUMBER?

Reply:
`INITIAL_PID_LIMIT` reads like "initial limit for pid number", so it applies despite the implementation is a simple array, a hashtable or a radix-tree. Simple names may last longer?

Reply:
The limit normally means a fixed min/max number, but the number will increase to bigger than INITIAL_PID_LIMIT here. If you don't want to expose the internal data structure, I think that PID_INITIAL_COUNT/PID_INITIAL_NUMBER is still better than xxx_LIMIT.

Reply:
Some of the `ulimit()` or `setrlimit()` are changeable on UNIX, so the word limit has already been used such a way. Maybe we can do the same?